target/linux/ramips/dts/mt7621_raisecom_msg1500-x.00.dts

```diff
@@ -0,0 +1,146 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "raisecom,msg1500-x.00", "mediatek,mt7621-soc";
```

Comment:
I'd prefer using minus instead of dot here as well, since using the dot should stay an exception:
"raisecom,msg1500-x-00"

Reply:
Thanks so much for your advice.